src/commands/asset-compute/test-worker.js

```diff
@@ -78,8 +78,21 @@ class TestWorkerCommand extends BaseCommand {
         //   asset-compute/
         //     workerA/
         //     workerB/
-
-        return fs.existsSync(TEST_DIR) ? fs.readdirSync(TEST_DIR) : [];
+        if (fs.existsSync(TEST_DIR)) {
+            const tests = fs.readdirSync(TEST_DIR);
+            // eslint-disable-next-line array-callback-return
+            return tests.filter(test => {
+                const testPath = path.resolve(TEST_DIR, test);
+                try {
+                    fs.readdirSync(testPath);
```

Comment:
Node has apis to to check for directories that doesn't require try/catch, for example `lstatSync(path).isDirectory()`.

But there is an even better way - from https://stackoverflow.com/a/24594123/2709:

```
fs.readdirSync(TEST_DIR, { withFileTypes: true })
    .filter(entry => entry.isDirectory())
    .map(entry => entry.name)
```